### ISSUE TITLE
"fictional" für Drehbuchvorlagen

### ISSUE THREAD

**Issue by nittka** [TVTower/Documentation#26]
* wird aktuell noch nicht ausgelesen
* sollte anders benannt sein
* Auswirkung insb. auf Produktionszeit (mehrere Takes pro Szene möglich...)
* wenn live dann automatisch "fictional"?

**Comment by GWRon** [comment]
> wenn live dann automatisch "fictional"?

Nein. Eine Live-Schalte vom Brandenburger Tor ist nicht notwendigerweise ausgedachter Natur.
Gleiches gilt fuer ein Konzert - wobei... kommt drauf an, wie nahe es sich an einem Theaterstueck/Musical orientiert.

Eventuell braucht es diese Unterscheidung jetzt auch noch gar nicht. Aber Grundgedanke war halt, ob hier mit Buehne/Kulisse/Kostuemen (im Sinne von Mittelalterkleidung usw - nicht "Abendgarderobe") usw gearbeitet werden muss.

Womoeglich kann man dies auch anders (mit einem Flag) ausgestalten oder einen Modifier reinbringen, der den "Aufwand" beschreibt (und unabhaengig vom "Inhalt" ist).
